book/ja/refs/DirectionalContactInteraction.md

```diff
@@ -0,0 +1,55 @@
+# DirectionalContactInteraction
+
+`DirectionalContact`相互作用は、結合が形成される粒子間の距離とその結合方向に対してかかる相互作用です。
+ポテンシャルは角度に関するポテンシャルと距離に関するポテンシャルの積として表現されます。
+
+粒子`i`番目,`j`番目,`k`番目,`l`番目の粒子について、`i`,`j`,`k`がなす角に`angle1`で指定したポテンシャルが、`j`,`k`,`l`がなす角に`angle2`で指定したポテンシャルが、`j`,`k`間の距離に`contact`で指定したポテンシャルがかかります。
+
+{% math %}
+U(r, \theta_1, \theta_2) = U_contact (r) U_{angle_1} (\theta_1) U_{angle_2} (\theta_2)
+{% math %}
+
+
+角度に関する利用可能なポテンシャルには以下のようなものがあります。
+
+- [Cosine](CosinePotential.md)
+
+距離に関する利用可能なポテンシャルには以下のようなものがあります。
+
+- [Gasussian](Gaussian.md)
+- [GoContact](GoContact.md)
+
+## 例
+
+```toml
+[[forcefields.local]]
+interaction        = "DirectionalContact"
+potentials.angle1  = "Cosine"
+potentials.angle2  = "Cosine"
+potentails.contact = "GoContact"
+topology           = "none"
+margin             = 0.5 # relative length to longest cutoff
+parameters         = [
+    # 各ポテンシャルそれぞれについて、要求されるパラメータを設定します。
+    {indices = [0, 1, 2, 3], angle1_params = { v0 = ... }, angle2_params = { v0 = ... }, contact_params = { v0 = ... }},
```

Comment:
Do we really need `_params` suffix in `parameters` section? It is already clear that parameters are defined here, isn't it? I think `angle1 = {v0 = ...}, angle2 = {v0 = ...}, contact = {...}` is okay. Since this interaction takes 3 distinct potential parameters, lines would be long. There is no space to put redundant suffix.